library/pom.xml

```diff
@@ -159,6 +159,12 @@
 			<version>1.2.2</version>
 			<scope>runtime</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.webjars</groupId>
+			<artifactId>jqplot</artifactId>
+			<version>1.0.8r1250</version>
```

Comment:
May 2013 dependency ... maybe doesn't matter ..... does 1.0.9.d96a669-1 version work?

Reply:
I will try and see. I suspect it would work fine.